operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -0,0 +1,76 @@
+[id="examine-images-for-vulnerabilities"]
+= Examining images for vulnerabilities
+include::modules/common-attributes.adoc[]
+:context: examine-images-for-vulnerabilities
+
+toc::[]
+
+[role="_abstract"]
+With {product-title} you can analyze images for vulnerabilities.
+Scanner analyzes all image layers to check for known vulnerabilities by comparing them with the Common Vulnerabilities and Exposures (CVEs) list.
+
+When Scanner finds any vulnerabilities, it:
+
+* Shows them in the xref:../operating/manage-vulnerabilities.adoc#vulnerability-management-view_acs-operating-manage-vulnerabilities[*Vulnerability Management*] view for detailed analysis.
+* Ranks vulnerabilities according to risk and highlights them in the RHACS portal for risk assessment.
+* Checks them against enabled xref:../operating/manage-security-policies.adoc#manage-security-policies[security policies].
```

Comment:
In the preview, this cross-reference produces a 404 not found error. Looking at the `examine-images` feature branch in the fork repository, `operating/manage-security-policies.adoc` does not exist:

```
openshift-docs (examine-images)$ ls operating/manage-security-policies.adoc
ls: cannot access 'operating/manage-security-policies.adoc': No such file or directory
```

Please update the cross-reference accordingly. Thanks!

Reply:
Thanks @pneedle-rh This xref is part of https://github.com/openshift/openshift-docs/pull/34138 
which includes references from https://github.com/openshift/openshift-docs/pull/33962 

So should we merge them in order?
1. https://github.com/openshift/openshift-docs/pull/33962 
2. https://github.com/openshift/openshift-docs/pull/34138 
3. (This PR) https://github.com/openshift/openshift-docs/pull/34199

Reply:
Now that the other two PRs have been merged, the xref works.